geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapred/InputFormatBaseAdapter.scala

```diff
@@ -17,12 +16,14 @@ object InputFormatBaseAdapter {
   }
 
   def setConnectorInfo15(job: JobConf, user: String, token: PasswordToken) = {
-    val method = classOf[AccumuloInputFormat].getMethod("setConnectorInfo", classOf[Job], classOf[String], classOf[PasswordToken])
+    val method = Class.forName("org.apache.accumulo.core.client.mapred.InputFormatBase")
+        .getMethod("setConnectorInfo", classOf[JobConf], classOf[String], classOf[AuthenticationToken])
     method.invoke(null, job, user, token)
   }
 
   def setConnectorInfo16(job: JobConf, user: String, token: PasswordToken) = {
-    val method = classOf[AccumuloInputFormat].getMethod("setConnectorInfo", classOf[Job], classOf[String], classOf[PasswordToken])
+    val method = classOf[AccumuloInputFormat]
+        .getMethod("setConnectorInfo", classOf[JobConf], classOf[String], classOf[AuthenticationToken])
```

Comment:
the args and method name are the same, but the base class is different.
I wasn't able to fully test this against accumulo 1.6 - i was able to verify that at least the setConnector and setZoos worked, but I didn't have an accumulo 1.6 to test against.